sample-input/sph-factors/assembly/sph-region-avg.py

```diff
@@ -0,0 +1,119 @@
+import matplotlib.pyplot as plt
```

Comment:
Great! Check to see if my latest commit does the trick for all of the input
files.

On Fri, Jan 8, 2016 at 9:54 AM, sam <notifications@github.com> wrote:

> In sample-input/sph-factors/assembly/sph-region-avg.py
> <https://github.com/mit-crpg/OpenMOC/pull/219#discussion_r49195111>:
>
> > @@ -0,0 +1,119 @@
> > +import matplotlib.pyplot as plt
>
> Yep, that fixes it!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mit-crpg/OpenMOC/pull/219/files#r49195111>.
>



-- 
Will Boyd
Nuclear Science & Engineering
Massachusetts Institute of Technology
Cell: 423.413.8469
wbinventor@gmail.com